chef_master/source/chef_license_accept.rst

```diff
@@ -0,0 +1,277 @@
+The MLSA applies to customers with a commercial contract with Chef Software, and the EULA covers all other cases.
+
+-----------------------------------------------------
+Accept the Chef MLSA
+-----------------------------------------------------
+There are three ways to accept the Chef MLSA:
+
+#. When running ``chef-<PRODUCT-NAME>-ctl reconfigure`` the Chef MLSA is printed. Type ``yes`` to accept it. Anything other than typing ``yes`` rejects the Chef MLSA, and the upgrade process will exit. Typing ``yes`` adds a ``.license.accepted`` file to the ``/var/opt/<PRODUCT-NAME>/`` directory. As long as this file exists in this directory, the Chef MLSA is accepted and the reconfigure process will not prompt for ``yes``.
+
+#. Run the ``chef-<PRODUCT-NAME>-ctl reconfigure`` command using the ``--accept-license`` option. This automatically types ``yes`` and skips printing the Chef MLSA.
+
+#. Add a ``.license.accepted`` file to the ``/var/opt/<PRODUCT-NAME>/`` directory. The contents of this file do not matter. As long as this file exists in this directory, the Chef MLSA is accepted and the reconfigure process will not prompt for ``yes``.
+
+-----------------------------------------------------
+ Accept the Chef EULA
+-----------------------------------------------------
+
+Workstation Products
+=====================================================
+* Chef Workstation, which also contains
+    * Chef Infra
+    * Chef InSpec
+    * Push Jobs Client
+* Chef Infra
+* Chef InSpec
+* Habitat
+
+These products are typically installed on a user's workstation.
+Two methods are generally used to accept the license for these products:
+
+#. ``--chef-license <value>`` argument passed to the command line invocation.
+
+#. ``CHEF_LICENSE=<value>`` as an environment variable.
+
+``<value>`` can be specified as one of the following:
+
+#. ``accept`` - Accepts the license and attempts to persist a marker file locally. Persisting these marker files means future invocations do not require accepting the license again.
+
+#. ``accept-silent`` - Similar to ``accept`` except no messaging is sent to STDOUT
+
+#. ``accept-no-persist`` - Similar to ``accept-silent`` except no marker file is persisted. Future invocation will require accepting the license again.
+
+If no command line argument or environment variable is set, these products will attempt to request acceptance through an interactive prompt.
+If the prompt cannot be displayed, then the product will fail with an exit code 172.
+
+If the product attempts to persist the accepted license and fails, a message will be sent to STDOUT, but product invocation will continue successfully. 
+In a future invocation, however, the license would need to be accepted again.
+
+Products with specific features or differences from this general behavior are documented below.
+
+Chef Workstation
+-----------------------------------------------------
+Chef Workstation contains multiple chef products.
```

Comment:
Isn't "chef" supposed to be formatted differently? "Chef" or "Chef Software"?

Reply:
Making this Chef Software would avoid confusion